### PR TITLE
PIP-963 fix and refactor get_attachment

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: accession/accession.py [hunk 1]
```diff
@@ -453,6 +453,25 @@ def file_has_qc(self, encode_file, qc_name):
             return True
         return False
 
+    def encode_attachment_data(self, data):
```
Comment (p-sud) [inline accession/accession.py]: Probably should make this a static method

File: accession/accession.py [hunk 2]
```diff
@@ -453,6 +453,25 @@ def file_has_qc(self, encode_file, qc_name):
...
+        """
+        Encodes the attachment data into a b64 datastring
+        input: data as bytes object
+        Output: data as string, encoded as b64
+        """
+        return b64encode(data).decode("utf-8")
+
+    def make_download_link(self, filename, extension):
```
Comment (p-sud) [inline accession/accession.py]: This should probably be a static method too

File: accession/accession.py (hunk 1 above)
Comment (ottojolanki) [reply accession/accession.py]: +1

File: accession/accession.py (hunk 2 above)
Comment (ottojolanki) [reply accession/accession.py]: +1

**Pull request merged**